dev/VSIX/ProjectTemplates/Desktop/CppWinRT/UnitTestApp/MainWindow.cpp

```diff
@@ -0,0 +1,19 @@
+#include "pch.h"
+#include "MainWindow.xaml.h"
+#if __has_include("MainWindow.g.cpp")
+#include "MainWindow.g.cpp"
+#endif
+
+using namespace winrt;
+using namespace Microsoft::UI::Xaml;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
```

Comment:
>http://aka.ms/winui-project-info [](http://example.com/codeflow?start=46&length=32)

we should chat with Steve White about adding some docs for the new C#, C++ unit test templates

Reply:
Yes. I have a task tracking getting good documentation for this.